server/requirements.txt

```diff
@@ -15,4 +15,5 @@ timm==0.6.13
 typeguard==2.13.3
 accelerate==0.18.0
 pytesseract==0.3.10
-gradio==3.24.1
\ No newline at end of file
+gradio==4.22.0
```

Comment:
**Suggestion**: Ensure that the updated version of "gradio" (4.22.0) is fully compatible with the project's current implementation. Review the "gradio" change logs and test extensively to identify any breaking changes or necessary adjustments in the codebase. <br> **Code Suggestion**: 
 ```
 +gradio==4.22.0 # Ensure compatibility with the project and test thoroughly.
 ```